.env.development.testing

```diff
@@ -4,7 +4,6 @@ APP_KEY=base64:RMlWJrRHIuHLYRplJvWaLNwWvRoVkQdDTZe6U1Z4xRY=
 APP_DEBUG=true
```

Comment:
please rename file back, we should keep two files in repository: `.env.development` and `.env.dev.testing`